mrtt-ui/src/components/ButtonSubmit.js

```diff
@@ -0,0 +1,16 @@
+import { Button } from '@mui/material'
+import PropTypes from 'prop-types'
+
+const ButtonSubmit = ({ isSubmitting }) => {
+  return (
+    <Button sx={{ marginTop: '1em' }} variant='contained' type='submit' disabled={isSubmitting}>
+      {isSubmitting ? 'Submitting...' : 'Submit'}
+    </Button>
+  )
+}
+
```

Comment:
Copied project details code into own component

Reply:
nice! good call on extracting this